stories/action-bar/action-bar.stories.js

```diff
@@ -43,8 +43,9 @@ export const backBtn = () => `<div class="fd-action-bar">
 backBtn.storyName = 'Default';
 backBtn.parameters = {
     docs: {
-        iframeHeight: 400,
-        storyDescription: 'The default action bar displays a page title, description, back button and action buttons. To display a back button, add the <code>fd-action-bar\\_\\_description--back</code> modifier class to the description element.'
+        description: {
+            story: 'The default action bar displays a page title, description, back button and action buttons. To display a back button, add the `fd-action-bar__description--back` modifier class to the description element.'
```

Comment:
Here is the only place where we have `docs => description => story` instead of `docs => storyDescription` i guess we should go with a unified approach across all the stories.

Btw, you have removed `iframeHeight: 400` i guess it should be reverted.

Reply:
1. `iframeHeight` doesn't work here. I think we need to. get rid of it.
2. i did that as an example how we should be moving toward. in Version 7.x `storyDescription` will be gone. Let's do that progressively.